packages/astro-feather-icons/README.md

```diff
@@ -1,45 +1,91 @@
-# astro-feather-icons
+# Astro Feather Icons
 
-[GitHub](https://github.com/edazpotato/astro-feather-icons) |
-[NPM](https://npmjs.com/package/astro-feather-icons)
+Feather is a collection of simply beautiful open source icons.
+Each icon is designed on a 24x24 grid with an emphasis on simplicity, consistency, and flexibility.
 
-## Install
+```shell
+npm install astro-feather-icons
+```
 
-```bash
-npm i astro-feather-icons --save
+```shell
 pnpm i astro-feather-icons
 ```
 
+```shell
+yarn add astro-feather-icons
+```
+
+[GitHub](https://github.com/edazpotato/astro-feather-icons) |
+[NPM](https://npmjs.com/package/astro-feather-icons)
+
+
+
 ## Usage
 
+All of the icons are available from a single import, and only the icons that are used will be added to the page.
+
 ```astro
 ---
-import { AirplayIcon, AtSignIcon, ... } from 'astro-feather-icons'
+import * as Icon from 'astro-feather-icons'
 ---
+<Icon.AirPlay />
+<Icon.AtSign />
+<Icon.Zap size="60" fill="gold" />
+```
+
+The icons are also available as individual imports.
 
-<AirplayIcon size="24" />
```

Comment:
Can we please keep this example in the `README.md`, as this is the way I like using Icons.

Reply:
Yes, of course.

Reply:
Done in 32d4d5920ec4625a79b1702021215c301f03d060